portfolio/src/main/webapp/script.js

```diff
@@ -12,11 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var slideIndex = 1;
+
+// Next/previous controls
```

Comment:
jsdoc https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

Reply:
done

Reply:
The format is:
`/** the documentation */`

https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

Reply:
ohh it's not showing up here but I did change the format in the new commits. Ty for the guide tho!